### PR TITLE
Add comments Statuses to data model documentation

### PR DESCRIPTION
**Pull request by dinaelhanan**
Fixes Part of #3331
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by dinaelhanan** [comment]
@publiclab/reviewers check this out please!

**Comment by sashadev-sky** [comment]
@dinaelhanan I cannot look at this right now but I added a label onto it to try and get it some attention :)

File: doc/DATA_MODEL.md [hunk 1]
```diff
@@ -72,6 +72,15 @@ They are typically cached for quick loading, and can be inserted anywhere in the
 
 Comments belong to Notes via `nid`, and each have an author via `uid`; primary key `cid`. Maps also have comments via `nid`, and Answers may also have comments, via `aid`.
 
+Node `status` -- a property of `comments`, can be:
```
Comment (gauravano) [inline doc/DATA_MODEL.md]:
```suggestion
Comment `status` -- a property of `comments`, can be:
```

File: doc/DATA_MODEL.md [hunk 2]
```diff
@@ -72,6 +72,15 @@ They are typically cached for quick loading, and can be inserted anywhere in the
...
+
+* 0: banned
+* 1: normal
+* 3: draft
+* 4: moderated -- i.e. node created by a first-time poster, and has not yet been "approved"
```
Comment (gauravano) [inline doc/DATA_MODEL.md]:
```suggestion
* 4: moderated -- i.e. comment created by a first-time poster, and has not yet been "approved"
```

File: doc/DATA_MODEL.md [hunk 3]
```diff
@@ -72,6 +72,15 @@ They are typically cached for quick loading, and can be inserted anywhere in the
...
+
+The status for Node is now the same for comment in our Code base except for status = 5
```
Comment (gauravano) [inline doc/DATA_MODEL.md]: How about removing this line? @dinaelhanan

**Comment by gauravano** [comment]
Some changes to do @dinaelhanan and nice work :tada:. Thanks!

File: doc/DATA_MODEL.md (hunk 3 above)
Comment (dinaelhanan) [reply doc/DATA_MODEL.md]: ok, will do. Thanks!

**Comment by dinaelhanan** [comment]
@gauravano done. Thanks!

File: doc/DATA_MODEL.md [hunk 4]
```diff
@@ -72,6 +72,13 @@ They are typically cached for quick loading, and can be inserted anywhere in the
 
 Comments belong to Notes via `nid`, and each have an author via `uid`; primary key `cid`. Maps also have comments via `nid`, and Answers may also have comments, via `aid`.
 
+Comment `status` -- a property of `comments`, can be:
+
+* 0: banned
+* 1: normal
+* 3: draft
```
Comment (gauravano) [inline doc/DATA_MODEL.md]:
```suggestion
```

I forgot to tell you about removing draft status too :neutral_face:.

Comment (gauravano) [reply doc/DATA_MODEL.md]: ok, I have made the change!

**Comment by gauravano** [comment]
Merged :100: :balloon:

**Pull request merged**